utils/run_nexus_hera.ksh

```diff
@@ -1,4 +1,7 @@
-#!/bin/ksh -x
+#!/bin/bash
```

Comment:
Should the file extension be changed as well?

Reply:
A good suggestion, it was updated